test/fft_utest.cpp

```diff
@@ -103,130 +103,6 @@ class FFTUtest {
         return vec;
     }
 
-    /**
```

Comment:
I think that this makes https://github.com/scality/nttec/issues/68 outdated, if so we could close it.